### PR TITLE
Add calculator job scenario for performance verification in B-001

### PR DESCRIPTION
**Pull request by dstenroejl**
- [x] Delete use of "categories" (attribute) on tests
- [x] Clean up service bus receiver code in test
- [x] Clean up configuration of analyzer rules for tests 
- [x] CalculatorJobScenario implemented

### CODE REVIEW

File: source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        public CalculationJobScenario(LazyFixtureFactory<CalculationJobScenarioFixture> lazyFixtureFactory)
+            : base(lazyFixtureFactory)
+        {
+        }
+
+        [ScenarioStep(0)]
+        [DomainFact]
+        public void Given_CalculationJobInput()
+        {
+            // TODO: Validate input (especially date)
```
Comment (JesperGomes) [inline source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs:41]: looks good to me

File: source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            using var assertionScope = new AssertionScope();
+            actualWaitResult.IsCompleted.Should().BeTrue();
+            actualWaitResult.Run.Should().NotBeNull();
+        }
+
+        [ScenarioStep(3)]
+        [DomainFact]
+        public void AndThen_CalculationJobDurationIsLessThanOrEqualToTimeLimit()
+        {
+            var calculationJobTimeLimit = TimeSpan.FromHours(4);
```
Comment (JesperGomes) [inline source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs:85]: not sure I understand the difference between this test and the previous one

Comment (JesperGomes) [reply source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs:85]: ...I think a comment about it is needed

File: source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs (hunk 1 above)
Comment (dstenroejl) [reply source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs:41]: Ok. Remove TODO.

File: source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs (hunk 2 above)
Comment (dstenroejl) [reply source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs:85]: First we focus on completing the calculation with a certain **wait** time. This is not an exact time for how long it took to perform the calculation, but the time it took for our **retry loop** to determine that the calculation was completed.

The next test verifies the **calculation job duration**, which is the time we want to reduce during our performance workshop.

Comment (JesperGomes) [reply source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs:84]: Ok. Would have been nice with an explanation that clarifies that as you just did.

Comment (JesperGomes) [reply source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs:84]: (In the code)

**Review by JesperGomes — DISMISSED** [review]

File: source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs (hunk 2 above)
Comment (dstenroejl) [reply source/dotnet/domain-tests/Features/Databricks/CalculationJobScenario.cs:93]: Done

**Review by JesperGomes — APPROVED** [review]

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**